### PR TITLE
[tools] Add subcase support for csv2xml and xml2csv 

### PR DESCRIPTION
**Pull request by haoxli**
- Add subcase support
- Rewrite csv2xml.py by object-oriented programming

### CODE REVIEW

**Comment by haoxli** [comment]
Pending for  xml2csv.py update.

File: tools/xml/csv2xml.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 # EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #
 # Authors:
-#         Liu, xin <xinx.liu@intel.com>
+#         Li, Hao<haox.li@intel.com>
```
Comment (zqzhang) [inline tools/xml/csv2xml.py]: Please add your name rather than replace.

Comment (haoxli) [inline tools/xml/csv2xml.py]: I write a new one, not update his script.

File: tools/xml/csv2xml.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 #   may be used to endorse or promote products derived from this work without
 #   specific prior written permission.
 #
-# THIS SOFTWARE IS PROVIDED BY INTEL CORPORATION "AS IS"
```
Comment (zqzhang) [inline tools/xml/csv2xml.py]: I just check the license in templates, which use `""`. Not sure where you get this update.

Comment (haoxli) [inline tools/xml/csv2xml.py]: I got it from stability tests: https://github.com/crosswalk-project/crosswalk-test-suite/blob/master/stability/stability-iterative-android-tests/iterative/Install_Uninstall_Repeatedly.py

Comment (zqzhang) [inline tools/xml/csv2xml.py]: That's not good. Please remove this change.

**Pull request merged**